jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -0,0 +1,142 @@
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir("${env.WORKSPACE}")
+
+    try {
+        sshagent(['openshift-bot']) {
+            advisory = params.ADVISORY ? Integer.parseInt(params.ADVISORY.toString()) : 0
+            def components = params.COMPONENTS.replaceAll(',', ' ').split(' ')
+            def filename = "tarball-srouce-output.txt"
+            def outdir = params.OUT_DIR.toString()
+            def rcm_guest = params.RCM_GUEST.toString()
+            currentBuild.description += "tarball: ${components}"
+
+            stage("elliott tarball-source") {
+                component_args = ""
+                components.each {
+                    component_args += "--component ${it}"
+                }
+                buildlib.elliott """
+                    tarball-sources create
+                    --out-dir=${outdir}
+                    --force
+                    ${component_args}
+                    ${advisory} > ${filename}
+                """
+            }
+
+            stage("rsync") {
+                cmd = "rsync -avz --chmod=go+rX ${outdir} ${rcm_guest}"
+                commonlib.shell(
+                    script: cmd,
+                    returnAll: true
+                )
+            }
+
+            stage("file ticket to RCM") {
+                cmd = """awk '/${advisory}/ { save=\$0 }END{ print save }' ${filename}"""
+                verify = commonlib.shell(
+                        script: cmd,
+                        returnStdout: true
+                )
+
+                commonlib.email(
+                    from: "aos-art-automation@redhat.com",
+                    to: "${params.MAIL_LIST_SUCCESS}",
+                    cc: "aos-team-art@redhat.com",
+                    replyTo: "aos-team-art@redhat.com",
+                    subject: "OCP Tarball sources",
+                    body: """
+                    Hi RCM,
+                    The OpenShift ART team needs to provide sources for `${components}` in https://errata.devel.redhat.com/advisory/${advisory}
+                    This release is urgent and we want it to be unblocked and shipped ASAP.
```

Comment:
Maybe this was the case in the ticket we listed as example, but I always remove that line ... wdyt ?

Reply:
You mean `This release is urgent and we want it to be unblocked and shipped ASAP.` ? Yes, I think most of the time it's not very urgent and doesn't block any other work.

Reply:
consider it a "fire and forget" thing. i do not think any work visible by us is blocked by this.